src/components/logo/Logo.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import shifraLogo from '../../images/shifraLogo.png';
+
+class SiteLogo extends React.Component {
+  render() {
+    return (
+        <img className="logo" src={shifraLogo} alt={this.props.alt} />
+      );
```

Comment:
The file name should match with the component. Also if you're going to hardcode the logo into the component, it's redundant to pass the alt text down via props. I would either just use "ShifraIO" directly onto the img, or pass everything down to the component and make it more reusable.